doc/functions/parent.rst

```diff
@@ -17,4 +17,22 @@ parent block when overriding a block by using the ``parent`` function:
 The ``parent()`` call will return the content of the ``sidebar`` block as
 defined in the ``base.html`` template.
 
-.. seealso:: :doc:`extends<../tags/extends>`, :doc:`block<../functions/block>`, :doc:`block<../tags/block>`
+``parent()`` can also be used in conjunction with the :doc:`block<../tags/use>` 
+tag, in which case it seems to pull from the most recently defined parent block: 
```

Comment:
nice addition! 
I would try to frase without the `seems to` part, as to me it looks like there is some uncertainty in the behavior of Twig while there isn't.